RUST/nstream-iter/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "nstream"
```

Comment:
Should I rename the packages to nstream{-iter,-unsafe}?

Reply:
I don't know that it matters.  I guess if the binary stays `nstream` then it's easier to script experiments since only the folder name will change.